Bluesky widget

jmedinaebi opened the pull request:
Adding Bluesky widget to IntAct Portal.

Draft PR as IntAct Bluesky account is currently empty as we haven't imported the history from twitter there yet, so I have used the EMBL-EBI account to test the widget. You can check it out here https://intact-portal.github.io/intact-portal-view/home

Because the sidebar in IntAct Portal is wider than in Complex Portal we don't need a max width, we can let the width on auto and adapt the width of the widget to the width of the side panel. I have added a min width so if the window is make smaller to the point that the sidebar is skinny we don't end up with a really skinny unreadable widget.

EliotRagueneau commented:
![Screenshot 2024-12-09 at 11 21 11](https://github.com/user-attachments/assets/efd3813a-617c-4fe8-b020-02be4937ccf2)
It does create quite a big gap bellow the main menu, and on a "small" screen in width, the skeets gets cropped, we should maybe remove again the icon to avoid that

EliotRagueneau commented:
Maybe we could afford to remove the latest news section. If I remember correctly, we added it to replace the tweets which were no longer working

jmedinaebi commented:
> Maybe we could afford to remove the latest news section. If I remember correctly, we added it to replace the tweets which were no longer working

I don't think it was added then, as it is actually not latest news, it's statistics about the latest release. We added latest news in IMEx but nowhere else. I'll take a look at updating margins and other stuff to see if I can make it look better.

jmedinaebi commented:
@EliotRagueneau I've made some style changes, including moving the newsletter section to the bottom. Please check how it looks now.
